lib/actions.js

```diff
@@ -35,30 +35,39 @@ var ACCOUNT_STATUS_ACTION = {
 
 // Actions that send an email, and hence might make
 // us look like spammers if abused.
-var EMAIL_SENDING_ACTION = {
+const EMAIL_SENDING_ACTION = {
   accountCreate: true,
   recoveryEmailResendCode: true,
   passwordForgotSendCode: true,
   passwordForgotResendCode: true,
   sendUnblockCode: true
 }
 
+// Actions that can send sms, and could make us
+// very annoying to a user if abused.
+const SMS_SENDING_ACTION = {
+  inviteUserSms: true
```

Comment:
I should have made a note about this. Since the auth-server sms endpoint will be generic, I needed some way to differentiate the type of sms being sent. Currently, this should only be for inviting users, however I could see us reusing the sms endpoint for 2FA etc. I am open for other another name.